zksync_sdk/wallet.py

```diff
@@ -317,25 +319,28 @@ async def transfer(self, to: str, amount: Decimal, token: TokenLike,
         transfer, eth_signature = await self.build_transfer(to, amount, token, fee, nonce, valid_from, valid_until)
         return await self.send_signed_transaction(transfer, eth_signature)
 
-    async def transfer_nft(self, to: str, nft: NFT, fee_token: TokenLike, 
-                          fee: Decimal = None, 
-                          valid_from=DEFAULT_VALID_FROM, 
-                          valid_until=DEFAULT_VALID_UNTIL
-                          ) -> List[TxEthSignature]:
+    async def transfer_nft(self, to: str, nft: NFT, fee_token: TokenLike,
+                           fee: Decimal = None,
+                           valid_from=DEFAULT_VALID_FROM,
+                           valid_until=DEFAULT_VALID_UNTIL
+                           ) -> List[Transaction]:
+        # -> List[TxEthSignature]: WHY THE TYPE WAS NOT TRANSACTION,
```

Comment:
> -> List[TxEthSignature]: WHY THE TYPE WAS NOT TRANSACTION,

Most likely a mistake from my side. It should definitely be list of transactions